### PR TITLE
Adding grant24 Planck Rev 6 keymap.

### PR DESCRIPTION
**Pull request by grant24**
Created custom keymap for the Planck centered around Colemak typing and Qwerty gaming. Details in the Readme.

## Description

Modified the default keymap for alphas and Adjust layer. Full custom keymap for all other layers (Raise, Lower, Fn, Mods)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/grant24/config.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#pragma once
```
Comment (drashna) [inline keyboards/planck/keymaps/grant24/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Comment (grant24) [reply keyboards/planck/keymaps/grant24/config.h:1]: On both the config.h and keymap.c, or just on the config.h?

**Review by drashna — APPROVED** [review]

File: keyboards/planck/keymaps/grant24/config.h [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                                }
+#endif
+
+/*
+ * MIDI options
+ */
+
+/* Prevent use of disabled MIDI features in the keymap */
+//#define MIDI_ENABLE_STRICT 1
+
```
Comment (fauxpark) [inline keyboards/planck/keymaps/grant24/config.h:36]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**